modules/creating-nfd-cr-cli-disconnected.adoc

```diff
@@ -0,0 +1,167 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-node-feature-discovery-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-nfd-cr-cli-disconnected_{context}"]
+= Create a NodeFeatureDiscovery CR by using the CLI in a disconnected environment
+
+As a cluster administrator, you can create a `NodeFeatureDiscovery` CR instance using the {oc-first}.
+
+.Prerequisites
+
+* You have access to an {product-title} cluster
+* You have installed the {oc-first}.
```

Comment:
```suggestion
* You installed the {oc-first}.
```
Revised to avoid the present perfect tense and use the simple past tense, per IBM Style.